internal/pkg/scaffold/gopkgtoml.go

```diff
@@ -50,31 +50,31 @@ required = [
 
 [[override]]
   name = "k8s.io/api"
-  # revision for tag "kubernetes-1.13.1"
-  revision = "05914d821849570fba9eacfb29466f2d8d3cd229"
+  # revision for tag "kubernetes-1.14.1"
+  revision = "6e4e0e4f393bf5e8bbff570acd13217aa5a770cd"
```

Comment:
Could we bump the version on these ones? 
```suggestion
 version = "kubernetes-1.14.1"
```